### PR TITLE
Update llvm-project to llvm/llvm-project@af20aff35ec3

### PR DESCRIPTION
**Pull request by hamptonm1**
LLVM Commit: af20aff35ec3
StableHLO Commit: 54dafb1c41

### CODE REVIEW

**Comment by hamptonm1** [comment]
FYI- @AlexandreEichenberger I upgraded to the latest version of LLVM/ StableHLO. So we should be up to speed now :)

**Review by AlexandreEichenberger — COMMENTED** [review]
Fantastic. I think the old commit string shows up in a few more places, do you mind updating it there too

image

**Comment by hamptonm1** [comment]
> Fantastic. I think the old commit string shows up in a few more places, do you mind updating it there too
> 
> image

@AlexandreEichenberger It is updated :)

File: utils/clone-mlir.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 git clone -n https://github.com/llvm/llvm-project.git
 # Check out a specific branch that is known to work with ONNX-MLIR.
-cd llvm-project && git checkout 01d233ff403823389f8480897e41aea84ecbb3d3 && cd ..
+cd llvm-project && git checkout af20aff35ec37ead88903bc3e44f6a81c5c9ca4e && cd ..
```
Comment (hamptonm1) [inline utils/clone-mlir.sh:3]: @AlexandreEichenberger Check here please

Comment (AlexandreEichenberger) [reply utils/clone-mlir.sh:3]: Sorry, I see it now :-)

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM, thanks for updating to the latest, much appreciated

**Pull request merged**